.github/ISSUE_TEMPLATE/Run Team

```diff
@@ -0,0 +1,12 @@
+**Technical specification**
```

Comment:
Maybe name the file "Adding an ABI" we could have more differents processes with the run team in the future.

I don't think this format will be handled by github you can take inspiration from existing ones : https://github.com/massalabs/massa/blob/c264a2a59729b38a598348f6d9f308b488c05a0f/.github/ISSUE_TEMPLATE/bug_report.md and the doc : https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository